package.json

```diff
@@ -10,6 +10,9 @@
             "js"
         ]
     },
+    "scripts": {
+        "test": "node node_modules/jest/bin/jest.js"
```

Comment:
why do we need this? tests used to be run from the /regression and /test directories. if we want to launch module-level tests, we can use workspaces.

Reply:
Was debating whether to keep this or not.  Removed.